releasetool.yaml

```diff
@@ -0,0 +1,88 @@
+---
+projects:
+  vars:
+    oses:
+      - rhel7
+    signer: packages@theforeman.org
+    tar_projects: []
+    rpm_packages: []
+    koji_tags: []
+    artifact_strategy: 'rake'
+    pass_name_gpg: "theforeman/releases/foreman/{{ foreman_version }}-gpg"
+    pass_name_key: "theforeman/releases/foreman/{{ foreman_version }}-key"
```

Comment:
Can this path be constructed internally from known variables? It seems predictable without having to hard-code.